server/backend/news.go

```diff
@@ -34,3 +36,39 @@ func (s *CampusServer) GetNewsSources(ctx context.Context, _ *emptypb.Empty) (ne
 	}
 	return &pb.NewsSourceReply{Sources: resp}, nil
 }
+
+func (s *CampusServer) GetNews(ctx context.Context, req *pb.GetNewsRequest) (newsSources *pb.GetNewsReply, err error) {
+	if err = s.checkDevice(ctx); err != nil {
+		return
+	}
+
+	var sources []model.News
+	// preloading files as the news images can be null
+	// I could not get outer joins to work => we are currently doing unnecessary db calls
+	tx := s.db.Preload("Files")
```

Comment:
nit: well this table could be really big eventually, so idk?

Reply:
Why is this always so difficult?
![image](https://github.com/TUM-Dev/Campus-Backend/assets/26258709/e141eb4b-056b-40e1-a9d4-c3c977ca450e)

You are correct => Raw query it is